besu/src/main/scripts/besu-entry.sh

```diff
@@ -45,5 +45,10 @@ done
 # Construct the command as a single string
 COMMAND="/opt/besu/bin/besu $@"
 
-# Switch to the besu user and execute the command
-exec su -s /bin/bash $BESU_USER_NAME -c "$COMMAND"
+# Check if current user is root
```

Comment:
Make sense, and probably if not root we can skip the all the part that fixes path permission, so I suggest to move this check at the very beginning of the script and if root just exec Besu, otherwise run the rest of the script, wdyt?

FYI @pullurib

Reply:
Yes, I agree, a check at very beginning of the script and if not root, just exec'ing Besu should be good.

Reply:
I've updated the PR and moved the user check to the beginning